setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup
+
+setup(
+    name="dune-sync",
+    version="0.1",
```

Comment:
Is this a dummy version which is not to be used? Otherwise, we could use the actual version number from the latest release.